pkg/syncer/metadatasyncer.go

```diff
@@ -1,12 +1,9 @@
 /*
 Copyright 2019 The Kubernetes Authors.
-
```

Comment:
keep same copyright template in all files. I see some lines are removed.

Reply:
Done

Reply:
Done